src/pykeen/nn/node_piece/cli.py

```diff
@@ -0,0 +1,105 @@
+"""Command-Line Interface for pre-computing tokenizations for NodePiece."""
+import logging
+import math
+import pathlib
+from typing import Optional
+
+import click
+import more_click
+
+from .anchor_search import anchor_searcher_resolver
+from .anchor_selection import anchor_selection_resolver
+from .loader import TorchPrecomputedTokenizerLoader
+from ...constants import PYKEEN_MODULE
+from ...datasets import dataset_resolver
+from ...datasets.utils import _digest_kwargs
+from ...utils import load_configuration
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+# TODO: no default results in error in class_resolver:
+# ValueError: no default given either from resolver or explicitly
+@dataset_resolver.get_option("-d", "--dataset", as_string=True, default="nations")
```

Comment:
you can add `required=True` to click options

Reply:
The problem seems to be the combination of `as_string` and no default.

Reply:
hmm sounds like a class_resolver issue to fix huh

Reply:
done in https://github.com/cthoyt/class-resolver/pull/38